package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "test": "mocha -r ts-node/register test/libraryUtilitiesTest.ts",
```

Comment:
@Benglin This is to include nested folders in "__tests__" folder.  I've fixed a bug in this line in last commit in this PR.